### PR TITLE
Create Entities of the diagram

### PR DESCRIPTION
**Pull request by Ygoraphael**

### CODE REVIEW

File: Portal_Perfomance_Employes/Models/Certification.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class Certification
+    {
+        [Key]
+        public int CertificationId { get; set; }
+        [ForeignKey("SkillId")]
+        public int SkillId { get; set; }
+        public Skill Skill { get; set; }
+        public string CertificationName { get; set; }
+        public ICollection<EmployeeCertification> EmployeeCertifications { get; set; } = new List<EmployeeCertification>();
```
Comment (dangoth) [inline Portal_Perfomance_Employes/Models/Certification.cs:14]: I don't think we should have the collection here, it should be enough to have it on the Employee

File: Portal_Perfomance_Employes/Models/EmployeeCertification.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace PortalPerfomanceEmployees.Models
+{
+    public class EmployeeCertification
+    {
+        [Key]
+        public int Id { get; set; }
+        [ForeignKey("CertificationId")]
+        public int CertificationId { get; set; }
+        public Certification Certification { get; set; }
```
Comment (dangoth) [inline Portal_Perfomance_Employes/Models/EmployeeCertification.cs:12]: I think we just need the CertificationId here, if we want the certification we can retrieve it by ID

File: Portal_Perfomance_Employes/Models/EmployeeCertification.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        [ForeignKey("Id")]
+        public int EmployeeId { get; set; }
+        public Employee Employee { get; set; }
```
Comment (dangoth) [inline Portal_Perfomance_Employes/Models/EmployeeCertification.cs:15]: same for employee

**Review by Tsarar — APPROVED** [review]

File: Portal_Perfomance_Employes/Models/EmployeeCertification.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        [DataType(DataType.Date)]
+        [DisplayFormat(DataFormatString = "{0:yyyy/MM/dd}", ApplyFormatInEditMode = true)]
+        public DateTime DateCertified { get; set; }
+        public string CertifcateUrl { get; set; }
```
Comment (dangoth) [inline Portal_Perfomance_Employes/Models/EmployeeCertification.cs:19]: Certificate

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]

**Review by VladyslavCherednychenko — APPROVED** [review]

**Review by dangoth — APPROVED** [review]

**Review by dariia-kachalova — APPROVED** [review]

**Pull request merged**